README.md

```diff
@@ -124,29 +127,47 @@ class MyComponent {
 }
 ```
 
-The HTTP will include the following Connection types, and will provide an interface to ensure third-
-party Connection implementations conform to the expected API.
+The Http library will include the following `Connection` factories, and will provide an interface to
+ensure third-party `Connection` implementations conform to the expected API.
 
- * XHRConnection
- * JSONPConnection
- * NodeConnection
- * MockConnection
+ * Http
+ * HttpJsonp
+ * HttpNode
+ * HttpMock
+ * _HttpFetch_ TBD
```

Comment:
The thought came to mind because of the hope that progress would be better with fetch. I've had some difficulty with experiments with progress+XHR (at least download progress).